static/home/js/gprotselectivitytree.js

```diff
@@ -164,7 +164,7 @@ for (var x in selectivityinfo){
     var spacer = 8
 
 
-    if(selectivityinfo[x].indexOf("Gs family") >= 0){
+    if(selectivityinfo[x].indexOf("Gs") >= 0){
```

Comment:
I know this is a legacy function, but maybe merge the four if-statements into a single for loop.